api/utils/keys/hardwaresigner_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build !linux || libpcsclite
+// +build !linux libpcsclite
```

Comment:
I don't think we need the old `build` directive. We require go 1.18 now anyways.